src/api/v1/db/oracledb/doughs-dao.js

```diff
@@ -117,37 +188,59 @@ const getDoughs = async (filters) => {
 };
 
 /**
- *
- * @param {Array} queries
+ * Generate bind params from a POST body, discarding invalid
+ * data attributes and adding default values for non-included attributes
+ * @param {object} body
+ * @returns {object} bindparams
+ * @throws {Error} when an attribute not in doughsProperties is found in body.
+ * It's assumed that body has already been checked for invalid properties meaning that when
+ * one is found here it's a server error rather than a user error.
+ */
+const getPostBindParams = (body) => {
+  const { attributes } = body.data;
+  const bindParams = doughsOutBindParams;
+  _.forEach(attributes, (attributeValue, attributeName) => {
+    if (!(attributeName in doughsProperties)) {
+      throw new Error('Invalid attribute found');
+    }
```

Comment:
I don't know if this code should trust 100% that the openapi doc will catch every possible invalid param; this seems like some good defense in depth to stop any errors before they hit the database

Reply:
That's probably fine for this case since the bind parameters are being generated programmatically. Usually these end up being hardcoded instead. Could you also update the documentation for this function?

Reply:
@ianbrown9475 what did you mean by updating the documentation? It already has an `@throws` note and I don't think the signature has changed. Sorry if I'm missing something.